README.md

```diff
@@ -39,4 +39,24 @@ The consumer will periodically run to seed a mongo database from the HackerNews
 
 The api reads data from a mongo database and returns the necessary information based on the API path.
 
+#### GRPC
 
+The GRPC service support communication between services
+
+#### Updating the generated go files
+
+If you update the `.proto` then you need to run the following command:
+
+```bash
+protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative pkg/grpc/proto/hackernews.proto 
```

Comment:
Minor: Might be worth a Makefile to simplify this? `make proto`

Reply:
Done